Zend/tests/enum/__set.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Enum __set
+--FILE--
+<?php
+
+enum Foo {
+    case Bar;
+
+    public function __set(string $name, mixed $value)
+    {
+        return '__set';
```

Comment:
`__set()` is a void method (https://wiki.php.net/rfc/magic-methods-signature#proposal), so there shouldn't be any return